actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -259,7 +223,7 @@ pub fn call<RT: Runtime>(
     .into();
 
     // copy return data to output region if it is non-zero
-    copy_to_memory(memory, output_offset, output_size, U256::zero(), &state.return_data)?;
+    copy_to_memory(memory, output_offset, output_size, U256::zero(), &state.return_data, false)?;
```

Comment:
I don't know what they were thinking here, but this is the _one_ case where the output buffer isn't zeroed.